### ISSUE TITLE
Implement W, B, and V key sequences.

### ISSUE THREAD

**Issue by stratact** [jugglerchris/textadept-vi#30]
First, let me express my appreciation for you making this module, it's awesome!
Second I'm very new to textadept, I just installed it today and still playing around it. I'm trying to set it up to the way I like it.

So I would like to have the normal command key sequences I mentioned in the title implemented because I use them a lot in Vim to traverse words with `W` and `B` a lot faster and I need `V` to select whole lines.

If you can implement this, it would make my life a whole lot easier :smile_cat:

**Comment by stratact** [comment]
Closing, due to lost interest.

**Comment by jugglerchris** [comment]
Hi,
Sorry for the slow reply (holidays)!
Despite the lost interest, I'm reopening as it's still a legitimate bug/feature request (depending on your point of view), and I aim to implement them at some point.  I don't think any of them are going to be too much work given that `w`, `b`, and `v` are already there.

Chris

**Comment by Nanoc-ice** [comment]
Just to add on to this request is the ability to change things with multiple lines.  For example I will comment out a function, I will set a mark  go to the end of the block and then do :.c’as/^/—/.  Or use the selection in V (line mode or Ctrl-V) and then shift-i.  I’m don’t know if you can make the kind of changes to the ‘:’ mode, but there is a lot of substitution regex stuff that can be accomplished.

**Comment by jugglerchris** [comment]
Hi @Nanoc-ice ,

Pretty much anything can be implemented, it's just a matter of effort and time.  :-)  I've prioritised the commands I use a lot, but I'm also interested in learning more!  It's easier for me (which means most likely to happen) if each feature request is in its own issue, though!

Thanks,

jugglerchris

**Comment by Nanoc-ice** [comment]
Yes, my bad.  I will open up another issue.  But in line with this issue, I would put a  +1 for, at least, V.
